com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/Editor/ShaderGraph/Targets/BuiltInLitSubTarget.cs

```diff
@@ -42,7 +45,8 @@ public NormalDropOffSpace normalDropOffSpace
         public override void Setup(ref TargetSetupContext context)
         {
             context.AddAssetDependency(kSourceCodeGuid, AssetCollection.Flags.SourceDependency);
-            //context.AddCustomEditorForRenderPipeline("ShaderGraph.PBRMasterGUI", typeof(UnityEngine.Rendering.Universal.UniversalRenderPipelineAsset)); // TODO: This should be owned by URP
+            if (!context.HasCustomEditorForRenderPipeline(null))
+                context.customEditorForRenderPipelines.Add((typeof(BuiltInLitGUI).FullName, ""));
```

Comment:
I have to manually add the custom editor here as there's no actual renderer pipeline object for built-in. The empty string seems to work to identify it though